tools/InstallationTests/AutoTestLogin.ps1

```diff
@@ -0,0 +1,490 @@
+﻿<#
+# ----------------------------------------------------------------------------------
+#
+# Copyright Microsoft Corporation
+# Licensed under the Apache License, Version 2.0 (the "License");
+# you may not use this file except in compliance with the License.
+# You may obtain a copy of the License at
+# http://www.apache.org/licenses/LICENSE-2.0
+# Unless required by applicable law or agreed to in writing, software
+# distributed under the License is distributed on an "AS IS" BASIS,
+# WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+# See the License for the specific language governing permissions and
+# limitations under the License.
+# ----------------------------------------------------------------------------------
+
+This script will allow to login as long as you have required certificate installed on your machine
+
+WorkFlow# 1
+===========
+1) Will Check if the certificate is installed
+2) If not will ask you to login (one time) to access KeyVault
+3) Download Certificate, install certificate on the machine (will prompt to set certificate password)
```

Comment:
Any prompts should be ShouldContinue prompts, so that they will fail when this is being run as part of script execution.  The idea is to allow extra bootstrapping as a convenience to manual users, but allowing automation to fail rapidly if the machine is not properly set up.

Also, just to be clear.  A properly set up machine should require no downloads from KeyVault, I think